lamin_cli/_save.py

```diff
@@ -56,8 +56,13 @@ def save_from_filepath_cli(
     auto_connect_state = ln_setup.settings.auto_connect
     ln_setup.settings.auto_connect = True
 
-    import lamindb as ln
-    from lamindb._finish import save_context_core
+    try:
+        import lamindb as ln
+        from lamindb._finish import save_context_core
+    except Exception as e:
```

Comment:
Ideally, we would catch the django exception but we can't do that without importing it right?

Reply:
Generally, this whole thing would likely better be placed into `lamindb_setup._check_setup`.

The error that you're addressing is another variant of the error that Sergei resolves here:

- https://github.com/laminlabs/lamindb-setup/pull/946

Sergei addresses the issue in which a module is imported but not part of the instance. This here addresses the issue of no instance being configured at all, right? 🤔

Reply:
Or am I wrong? 🤔

Reply:
I had been using and testing `if not ln_setup._check_instance_setup()` for a while but it didn't seem to work in all cases. I can play with that again and provide the details.

Reply:
> No, i don't really see how this can be done in a more general way in _check_instance_setup as we can't really raise an exception there if instance is not configured

I think you're wrong, @Koncopd.

If you try to `import lamindb` with `auto_connect` `True` without an instance being configured, then `_check_instance_setup(from_module="lamindb")` can simply throw an error (I'm surprised it doesn't do that yet).

But this PR here is so surgical that we can easily refactor and it's not a big deal if I'm indeed right.

Great that you cleaned this up and arrived at this simple solution! 😄